app/views_redesigned/ad_templates/@responsive_footer/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<% cache_if @campaign.persisted? && @creative.persisted?, [@campaign, @creative] do %>
+  <div id="cf">
+    <a data-href="campaign_url" class="cf-img-wrapper" target="_blank" rel="nofollow noopener">
```

Comment:
This is a change being ported over to the ad_templates in our current views folder but can you do us a favor and change `nofollow` to `sponsored` in each of the files under `app/views_redesigned/ad_templates/`?

Reply:
Sure! I saw a related PR, just want to double check this wasnt finished up. And I'm not seeing a `ad_templates/` directory under master. Do you want me to copy that over?

Reply:
:heavy_check_mark: